otx/algorithms/classification/configs/configuration.yaml

```diff
@@ -10,7 +10,7 @@ learning_parameters:
       stable. A larger batch size has higher memory requirements.
     editable: true
     header: Batch size
-    max_value: 512
+    max_value: 2048
```

Comment:
Why should we change the max batch size in this PR? is it related to transformer?

Reply:
Yes, that's right. It says we need to raise that max value to support the VisionTransformer-large model. (report from @JihwanEom)

Reply:
Not limited for ViTs, 512+ batch size does not work on current develop..

Reply:
Actually, I'd like to know why these min/max values are in config (batch size, lr, ...).
I think there are some cases that might require values that are out of the range, and in these cases, these restricted ranges are inconvenient and too complicated to handle from the user's point of view.

Reply:
I'm guessing that's probably an attribute for the geti UI? I agree with Sungchul's comment. But this is probably tied to geti. We need to check that first.